### PR TITLE
Upload task to rollup storage

### PR DESCRIPTION
**Pull request by tolak**
This PR contains the following tasks:

- [x] Setup initial worker accounts to rollup storage
- [x] Upload task to rollup storage
- [x] Initialize task (e.g. apply worker account and nonce)

### CODE REVIEW

**Comment by tolak** [comment]
This PR contains some work of fetching nonce from the EVM chain, we have a dedicated PR #36 to implement fetch nonce from sub-chains and balances query

File: contracts/index_executor/src/lib.rs [hunk 1]
```diff
@@ -183,6 +184,51 @@ mod index_executor {
             Ok(())
         }
 
+        #[ink(message)]
+        pub fn setup_worker_accounts(&mut self) -> Result<()> {
```
Comment (h4x3rotab) [inline contracts/index_executor/src/lib.rs:188]: Why is it mutable (tx)?

File: contracts/index_executor/src/lib.rs [hunk 2]
```diff
@@ -183,6 +184,51 @@ mod index_executor {
...
+            let config = self.ensure_configured()?;
+            // Get rpc info from registry
+            let chain = self
+                .get_chain("Khala".to_string())
+                .map(Ok)
```
Comment (h4x3rotab) [inline contracts/index_executor/src/lib.rs:193]: You may want ".ok_or()?"

**Review by h4x3rotab — COMMENTED** [review]
Minor problems. Almost perfect!

**Pull request merged**